worlds/generic/docs/commands_en.md

```diff
@@ -1,96 +1,124 @@
-are always submitted to the server prepended with an exclamation point: `!`.
+In clients which have their own commands the commands are typically prepended by a forward slash: `/`. 
 
-#### Local Commands
+Server commands are always submitted to the server prepended with an exclamation point: `!`. <br/>
 
-The following list is a list of client commands which may be available to you through your Archipelago client. You
+# Server Commands
+
+Server commands may be executed by any client which allows for sending text chat to the Archipelago server. If your
+client does not allow for sending chat then you may connect to your game slot with the TextClient which comes with the
+Archipelago installation. In order to execute the command you need to merely send a text message with the command,
+including the exclamation point.
+
+### General
+- `!help` Returns a listing of available commands.
+- `!license` Returns the software licensing information.
+- `!options` Returns the current server options, including password in plaintext.
+- `!players` Returns info about the currently connected and non-connected players.
+
+### Utilities
+- `!countdown <number of seconds>` Starts a countdown using the given seconds value. Useful for synchronizing starts.
+  Defaults to 10 seconds if no argument is provided.
+- `!alias <alias>` Sets your alias, which allows you to use commands with the alias rather than your provided name.
+- `!admin <command>` Executes a command as if you typed it into the server console. Remote administration must be
+  enabled.
+
+### Information
+- `!remaining` Lists the items remaining in your game, but not where they are or who they go to.
+- `!missing` Lists the location checks you are missing from the server's perspective.
+- `!checked` Lists all the location checks you've done from the server's perspective.
+
+### Hints
+- `!hint` Lists all hints relevant to your world, the number of points you have for hints, and how much a hint costs.
+- `!hint <item name>` Tells you the game world and location your item is in, uses points earned from completing locations.
+- `!hint_location <location>` Tells you what item is in a specific location, uses points earned from completing locations.
+
+### Collect/Release
+- `!collect` Grants you all the remaining items for your world by collecting them from all games. Typically used after 
+goal completion.
+- `!release` Releases all items contained in your world to other worlds. Typically, done automatically by the sever, but
+can be configured to allow/require manual usage of this command.
+
+### Cheats
+- `!getitem <item>` Cheats an item, if it is enabled in the server.
+
+### Broken/Deprecated/WIP
+- `!status` Returns information about your team. <br /> (Currently all players as teams are unimplemented. Instead,
+this shows the connection status and check completion numbers for all players in the current room.)
```

Comment:
I feel like it shouldn't mention being Broken/Deprecated and just point out what it does do.
```suggestion
### Game status
- `!status` Returns information about the connection status and check completion numbers for all players in the current room. <br /> (Optionally mention a Tag name and get information on who has that Tag. For example: !status DeathLink)
```